src/discovery/discovery.rs

```diff
@@ -940,7 +940,7 @@ impl Discovery {
   pub fn handle_subscription_reader(&mut self, read_history: Option<GuidPrefix>) {
     let drds: Vec<std::result::Result<DiscoveredReaderData, GUID>> =
       match self.dcps_subscription_reader.read(
-        std::usize::MAX,
+        usize::MAX,
```

Comment:
Use the new associated const (added in 1.43)
https://doc.rust-lang.org/std/primitive.usize.html#associatedconstant.MAX
https://doc.rust-lang.org/std/usize/constant.MAX.html